recipes/azure-mgmt-core/LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
Please remove this copy of the license file since one is included in the source distribution as `LICENSE.md`.

Reply:
I havbe updated this part